### DISCUSSION TITLE
Should I Include @tailwind Directive Across Multiple CSS Modules?

### DISCUSSION THREAD

**Discussion by manavm1990** [tailwindlabs/tailwindcss#8709] [General]
In React, if using CSS modules across multiple components is it appropriate to repeat `@tailwind components` directive in each of these files such that we then have:

```css
@tailwind components

@layer components {
  .MyComponents {
    @apply text-center;
  }
}
```

I guess it is fine - just making sure and seeing 👀 how others might be doing this across CSS modules.

**Comment by rimzzlabs** [comment]
Hello there, I've been using css module some time and you don't need to declare the @tailwind directive again on each module
